src/project-tests/tribute-page-tests.js

```diff
@@ -9,154 +10,185 @@ export default function createTributePageTests() {
 
   describe('#Tribute Page tests', function() {
 
+    describe('#Technology Stack', function() {
+      it(responsiveWebDesignStack, function() {
+        assert.ok(true);
+      });
+    });
+
     describe('#Content', function() {
-      it(`1. My tribute page should have an element with corresponding
-        id="main", which contains all other elements.`, function() {
-          FCC_Global.assert.isNotNull(document.getElementById('main'));
-          FCC_Global.assert(
-            document.querySelectorAll('#main div, #main a, #main h1, #main img')
-            .length,
-            'element with id="main" must contain other elements');
+      let reqNum = 0;
+
+      reqNum++;
```

Comment:
Sorry to bug on this, but were you going to do a mocha beforeEach() for incrementing reqNum ?